src/chainstate/stacks/address.rs

```diff
@@ -206,6 +207,7 @@ impl StacksAddress {
     }
 
     pub fn to_b58(self) -> String {
+        increment_stx_addresses_created(); //promserver
```

Comment:
The method `to_b58()` is used in debugging and would get called all over the place, so this is probably isn't a good place for this either.  What are you trying to track here?